### DISCUSSION TITLE
✅ `supervision-0.12.0` checklist

### DISCUSSION THREAD

**Discussion by SkalskiP** [roboflow/supervision#181] [Contribution]
## overview

Hi everyone 👋🏻! The `supervision-0.12.0` release will be our biggest to date. Never before have we had so many external contributors working simultaneously. 🔥 It is a great joy but also a challenge. To make it easier (mainly for me) to manage the release work, I created a centralized list of tasks we plan to complete.

## notes

- All issues and pull requests related to `supervision-0.12.0` release got marked with `version: 0.12.0` label.
- Please be patient. 🙏🏻 I love to work on `supervision`, but I have many other responsibilities. I always try to be as responsive as possible.
- My primary `supervision` objective is to close all issues and pull requests related to `supervision-0.12.0`.

## checklist

- [x] phasing out Python `3.7` support by @onuralpszr
   - [x] https://github.com/roboflow/supervision/issues/179
   - [x] https://github.com/roboflow/supervision/pull/180
- [x] migrate from `setup.py` to `project.toml` by @onuralpszr
   - [x] https://github.com/roboflow/supervision/pull/53
   - [x] https://github.com/roboflow/supervision/pull/180
- [x] allow installing `supervision` in headless and desktop mode by @onuralpszr
   - [x] https://github.com/roboflow/supervision/issues/130
- [x] model evaluation using confusion matrix by @kirilllzaitsev
   - [x] https://github.com/roboflow/supervision/issues/140
   - [x] https://github.com/roboflow/supervision/pull/177
- [x] fix coco dataset loading bugs by @hardikdava and @SkalskiP 
   - [x] https://github.com/roboflow/supervision/pull/176
   - [x] https://github.com/roboflow/supervision/pull/169
- [x] mmdetection connector by @hardikdava
   - [x] https://github.com/roboflow/supervision/pull/173
- [ ] annotators redesign by @SkalskiP 
   - [ ] https://github.com/roboflow/supervision/pull/170
   - [ ] unicode label support by @Ucag
      - [ ] https://github.com/roboflow/supervision/issues/143
      - [x] https://github.com/roboflow/supervision/pull/159
   - [ ] advanced box annotator by @hardikdava
      - [ ] https://github.com/roboflow/supervision/issues/168
      - [x] https://github.com/roboflow/supervision/pull/171
   - [ ] tracking annotator by @hardikdava
      - [ ] https://github.com/roboflow/supervision/issues/49

**Comment by SkalskiP** [comment]
@hardikdava let's focus on closing those three PR first:
- https://github.com/roboflow/supervision/pull/176
- https://github.com/roboflow/supervision/pull/169
- https://github.com/roboflow/supervision/pull/173

Looks like it should be reasonably easy to cross those off our list. 🔥

**Reply by hardikdava** [reply]
@SkalskiP #173 and (#171 and #170) are ready for tests.

**Comment by hardikdava** [comment]
@SkalskiP how can I help to speed up everything?

**Comment by SkalskiP** [comment]
Hi @hardikdava, @onuralpszr, @kirilllzaitsev, @Ucag 👋🏻 I am officially wrapping up work on `supervision-0.12.0`. We are releasing it tomorrow at 11.00 AM. I'll drop posts on Twitter and LinkedIn and tag you all, assuming I know your accounts :)

Fantastic work, everyone! It was a pleasure. 🔥 I'm learning so much along the way.

I apologize for the lack of time on my side. I know I was blocking work in some cases. I'll try to do a better job with `supervision-0.13.0`. 🙏🏻

**Reply by onuralpszr** [reply]
Thank you all for awesome work, it was team efforts, and special thanks to @SkalskiP to gather us all and I also personally want say thank you for invite to project I hope it bring more usefulness as,well

**Reply by hardikdava** [reply]
Thanks everyone, we all are contributing with our specific skills which is admirable and very good management thanks to @SkalskiP .
